### PR TITLE
added dark mode switch

### PR DESCRIPTION
**Pull request by ADHIRAJ-12**
hey @amanrajrana I've added the dark theme for the calculator.
Can you please review it and consider it as a contribution in hacktoberfest.
Thanks.
Screenshot (83)
Screenshot (82)

### CODE REVIEW

**Comment by amanrajrana** [comment]
@ADHIRAJ-12 Can you change the light theme background colour to `white`?

**Comment by ADHIRAJ-12** [comment]
Ya sure ! let me try

File: src/index.css [hunk 1]
```diff
@@ -2,3 +2,31 @@
 @tailwind components;
 @tailwind utilities;
```
Comment (ADHIRAJ-12) [inline src/index.css:4]: added hex code for white #FFFFFF.

**Comment by ADHIRAJ-12** [comment]
Attaching ss with white background.


Screenshot (84)

**Comment by ADHIRAJ-12** [comment]
@amanrajrana initially i added the toggle switch to the navbar itself, but somehow it was not switching themes brother.
if you can help me with this ,i can try brother.

**Comment by ADHIRAJ-12** [comment]
Update: @amanrajrana i have made a PR separately with the final code. please review that so that we can close this one.

**Pull request closed without merging**